matRadGUI.m

```diff
@@ -3113,27 +3109,54 @@ function SaveResultToGUI(~, ~, ~)
     dose = resultGUI.(handles.SelectedDisplayOption);
 end
 
-
-
+minDoseValOld      = handles.minDoseVal;
 maxDoseValOld      = handles.maxDoseVal;
-handles.maxDoseVal = max(dose(:));
+if ~isempty(handles.vRangeDose)
+    handles.minDoseVal = handles.vRangeDose(1);
+    handles.maxDoseVal = handles.vRangeDose(2);
+end
+
+% if upper colorrange is set to a specific value then use this value as upper threshold - otherwise 120% iso dose 
+% line will be plotted
+ upperMargin = 1;
+if abs((max(dose(:)) - handles.maxDoseVal)) < 0.01  * max(dose(:))
+    upperMargin = 1.2;
+end
```

Comment:
Why do we need the upper Margin right here? We have min and max dose value and a specified dose range. These defaults should only be defined in matRad_computeIsoDoseContours?!